src/MigrationTools/_EngineV1/DataContracts/WorkItemData.cs

```diff
@@ -22,31 +22,31 @@ public string Id
                     return _id;
                 }
 
-                _id = GetField("ID")?.ToString();
+                _id = GetField("System.Id")?.ToString(); // ToString() will create a new string while the field value already is a string => as cast is more efficient here
```

Comment:
With `(int)GetField("System.Id")` this line failes as "cant cast from `int` to `string` error. :(

Reply:
Ok.. never happened to me but then safety over efficiency of course..